Samples/SessionReplay-CameraTest/Config/SessionReplay-CameraTest.xcconfig

```diff
@@ -0,0 +1,27 @@
+#include "Architectures.xcconfig"
```

Comment:
`m`: Why do we need to create so many different config files instead of having it in one?

Reply:
one thing this helps with is scoping the git history of changes for a logical grouping of settings (as they are grouped in Xcode). and i suppose, just keeping the requirements on one's working memory manageable.

everything else can be done either way. for instance, one danger is adding the same setting multiple times with different values and not realizing it. this is easy to do in a single long list, and in different files. but a lint script could be written to check either kind of structure. so i don't see this as a motivating factor one way or the other.

Reply:
Okay while it is still a lot of files to look into I see your reasoning.